hack/make-rules/test.sh

```diff
@@ -32,6 +32,8 @@ export KUBE_CACHE_MUTATION_DETECTOR
 KUBE_PANIC_WATCH_DECODE_ERROR="${KUBE_PANIC_WATCH_DECODE_ERROR:-true}"
 export KUBE_PANIC_WATCH_DECODE_ERROR
 
+export KUBERNETES_MASTER="http://127.0.0.1:8080"
```

Comment:
needing to change an envvar that applies to all tests to keep tests passing smells like we're making an incompatible change... what happens if we don't set this?

Reply:
Only these 2 tests fail https://github.com/kubernetes/kubernetes/blob/2016fab3085562b4132e6d3774b6ded5ba9939fd/staging/src/k8s.io/kubectl/pkg/cmd/cmd_test.go#L74-L84 when we don't set this env variable. Maybe we can manually generate a dummy kubeconfig to overcome this issue instead of injecting this environment variable.

Reply:
Updated.